src/components/Contributors/styles.module.css

```diff
@@ -0,0 +1,54 @@
+.contributors {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.section_padding {
+    padding: 4rem 6rem;
+}
+
+.uptane_contributors {
+    width: 100%;
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 2rem;
+}
+
+.uptane_contributors h1 {
+    font-weight: 600;
+    font-size: 40px;
+    line-height: 55px;
+    color: var(--ifm-color-primary);
+}
+
+.contributor_list {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    gap: 2em;
+    padding: 0;
+}
+
+.contributor {
+    height: 70px;
+    width: 70px;
+    border-radius: 50%;
+    border: 2px solid #81dc7b;
```

Comment:
```suggestion
    border: 2px solid var(--ifm-color-primary);
```

Reply:
change the border color to the primary color as it matches with the theme of site, else LGTM